<launch>
	<arg name="uav_id" default="1"/>
    <arg name="uav_name" default="uav" />
    <arg name="demo_id" default="1" />
    
	<node if="$(eval demo_id == 1)" pkg="sunray_tutorial" type="takeoff_hover_land" name="takeoff_hover_land_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
	</node>

	<node if="$(eval demo_id == 2)" pkg="sunray_tutorial" type="block_xyzpos" name="block_xyzpos_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
	</node>

	<node if="$(eval demo_id == 3)" pkg="sunray_tutorial" type="circle_xyzvel" name="circle_xyzvel_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
	</node>

	<node if="$(eval demo_id == 4)" pkg="sunray_tutorial" type="circle_xyvelzpos" name="circle_xyvelzpos_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
	</node>

	<node if="$(eval demo_id == 5)" pkg="sunray_tutorial" type="hexagon_xyzposyawbody" name="pos_body_hexagon_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
	</node>

	<node if="$(eval demo_id == 6)" pkg="sunray_tutorial" type="vel_body_follow_car" name="vel_body_follow_car_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
	</node>

	<node if="$(eval demo_id == 7)" pkg="sunray_tutorial" type="vel_body_z_pos_follow_car" name="vel_body_z_pos_follow_car_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
	</node>
	
	<!-- <node pkg="sunray_tutorial" type="follow_car_xyvelzposyawbody" name="follow_a_car_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
	</node> -->
</launch>

